### ISSUE TITLE
Structured data products wrong object type for brand? - warning in search console

### ISSUE THREAD

**Issue by gregjotau** [Shopify/dawn#727]
https://schema.org/Brand

![Screenshot 2021-10-02 at 22 48 57](https://user-images.githubusercontent.com/13047497/135731411-5c5996bb-d017-4f43-8b72-11913fd2dc42.png)

Is my understanding correct that the brand most likely should be of type "Organization"?

![Screenshot 2021-10-02 at 22 50 19](https://user-images.githubusercontent.com/13047497/135731441-f028442c-62e3-46ab-b337-20fcfb88ca98.png)

**Comment by bakura10** [comment]
This seems to be new, as we've got this error very recently on our theme as well.

The most appropriate type though seems to be simply Brand:

```
"brand": {
      "@type": "Brand",
      "name": {{ product.vendor | json }}
    }
```

I have checked on one of our themes and this fixed the issue.

Google constantly changing their mind on how data should be formatted really shows that this should be handled at Shopify level and not at the theme level. This happens nearly every year, with Google suddenly making some types obsolete, some new information required... and so on.
